src/handlebars/about.handlebars

```diff
@@ -7,53 +7,81 @@
+
+          <ol>
+            <li>Provide a reliable source of OpenJDK binaries for all platforms, for the long term future!</li>
+            <li>Enable R&amp;D by OpenJDK developers, academics and researchers.</li>
+            <li>Provide an open, common, audited, build infrastructure for vendors to use (if they wish).</li>
+            <li>Provide a place to try out build and test infrastructure ideas that could be co-opted / re-implemented in <a target="_blank" href="https://openjdk.java.net">OpenJDK</a> and <a target="_blank" href="https://www.eclipse.org/openj9/">Eclipse OpenJ9</a>.</li>
+          </ol>
         </p>
+        </div>
       </div>
 
-      <div>
-        <h2 class="bold">About the AdoptOpenJDK Community</h2>
+      <div class="margin-bottom">
+        <h2 class="bold">Who's behind the AdoptOpenJDK.net Community?</h2>
         <p>
-          <a href="https://community.oracle.com/community/java/jcp/adopt-openjdk">AdoptOpenJDK</a> is a community of Java user group members, Java developers and vendors who are
-          advocates of <a href="https://openjdk.java.net">OpenJDK</a>, the open source project which forms the basis of the Java programming language and
-          platform.
+          <a href="https://community.oracle.com/community/java/jcp/adopt-openjdk">AdoptOpenJDK</a> is a
+          community of Java User Group (JUG) members, Java developers and vendors (including GoDaddy, IBM, jClarity, Microsoft, Pivotal and Red Hat)
+          who are advocates of <a target="_blank" href="https://openjdk.java.net">OpenJDK</a>, the open source
+          project which forms the basis of the Java programming language and
+          platform. The group also has other activities which you can discover in
+          the <a href="https://openjdk.java.net/groups/adoption/">Adoption Group</a>
+          at <a target="_blank" href="https://openjdk.java.net">OpenJDK</a>.
         </p>
         <p>
-          Part of the group's activities occur at the <a href="https://openjdk.java.net/groups/adoption/">Adoption Group</a> on the <a href="https://openjdk.java.net">OpenJDK</a> project.
+          See our <a href="sponsors.html">Sponsors</a> page for a subset of the vendors that are involved.
         </p>
       </div>
 
       <div class="margin-bottom">
         <h2 class="bold">License(s)</h2>
+          <p>
+          <ul>
+            <li>Build scripts and other code to produce the binaries, the website and other build infrastructure are licensed under
+              <a target="_blank" href="https://www.apache.org/licenses/LICENSE-2.0">Apache License, Version 2.0</a>.</li>
+            <li>OpenJDK code itself is licensed under <a target="_blank" href="https://openjdk.java.net/legal/gplv2+ce.html">GPL v2 with Classpath Exception</a>.</li>
+            <li>Eclipse OpenJ9 is licensed under <a target="_blank" href="https://github.com/eclipse/openj9/blob/master/LICENSE">Several licenses</a>.</li>
```

Comment:
This license list is confused. 
Build scripts etc - agreed.
OpenJDK source project has a variety of licenses, in the same sense as Eclipse OpenJ9 has a variety in the next line.  However, I think it is more useful to describe the license of the combined works binaries we distribute (i.e. GPLv2+CE)